frontend/app/src/components/dashboard/edit-snapshot/EditSnapshotDialog.vue

```diff
@@ -149,11 +150,29 @@ const updateAndComplete = (event: LocationDataSnapshot[]) => {
   updateLocationDataSnapshot(event);
   finish();
 };
+
+const steps = computed(() => {
+  const stepVal = get(step);
+  return [
+    {
+      title: t('dashboard.snapshot.edit.dialog.balances.title'),
+      state: stepVal > 0 ? StepperState.active : StepperState.inactive
+    },
+    {
+      title: t('dashboard.snapshot.edit.dialog.location_data.title'),
+      state: stepVal > 1 ? StepperState.active : StepperState.inactive
+    },
+    {
+      title: t('common.total'),
+      state: stepVal > 2 ? StepperState.active : StepperState.inactive
+    }
+  ];
+});
```

Comment:
agree, this should move to the lib...

think I did it the way it currently is, because of other states that do not depend on the step...

for instance, error state, a step can be active and still have error, based on our designs.

but since we're mostly focused on active/inactive/done, then we should take it to the lib.

Reply:
updated... needs https://github.com/rotki/ui-library-compat/pull/99